$OpenBSD: patch-man_meson_build,v 1.1 2017/10/30 10:27:42 ajacoutot Exp $

../gnome-color-manager-3.26.0/man/gcm-picker.sgml:1: parser error : StartTag: invalid element name
<!doctype refentry PUBLIC "-//OASIS//DTD DocBook V4.1//EN" [

Index: man/meson.build
--- man/meson.build.orig
+++ man/meson.build
@@ -1,4 +1,4 @@
-docbook2man = find_program('docbook2man', required : false)
+docbook2man = find_program('disable-docbook2man', required : false)
 if docbook2man.found()
   custom_target('gcm-calibrate-man',
     output : 'gcm-calibrate.1',
